### PR TITLE
External UCS parameters: Script 018

### PR DESCRIPTION
**Pull request by dboltovskyi**
Script is part of test set: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1169

### CODE REVIEW

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+-- [entityType: <Integer>, entityId: <Integer>]” -> of "<functional grouping>"
+-- -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as valid (with the pre-conditions of all other valid PTU content)
+-- b. add this “disallowed_by_external_consent_entities_off:
+-- [entityType: <Integer>, entityId: <Integer>]” field with the corresponding value
+-- to the corresponding "<functional grouping>" to the Policies database.
+--
+-- Preconditions:
+-- 1. Start SDL (make sure 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT)
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @dboltovskyi, please add step to check that 'disallowed_by_external_consent_entities_off' section is omitted in PreloadedPT

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+--
+-- Steps:
+-- 1. Register app1
+-- 3. Activate app1
+-- 4. Perform PTU (make sure 'disallowed_by_external_consent_entities_off' section is defined)
+-- 5. Verify status of update
+-- 6. Register app2
+-- 7. Activate app2
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @dboltovskyi, this step is redundant when device is consented. Only registering of second app will trigger PTU.

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+function Test:StartSession()
+  testCasesForExternalUCS.startSession(self, 2)
+end
+
+function Test:RAI_2()
+  testCasesForExternalUCS.registerApp(self, 2)
+end
+
+function Test:ActivateApp_2()
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @dboltovskyi, as device is consented to trigger PTU is enough only to register second app.
You can see "[PolicyTableUpdate] Request PTU - an app registered is not listed in PT (device consented)"

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua (hunk 1 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @istoimenova, Accepted

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua (hunk 2 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @istoimenova, Agree, but in script activation is linked with PTU and required in order to have table from snapshot created

File: test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua (hunk 3 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/018_ATF_P_TC_PTU_with_disallowed_by_external_consent_entities_off_struct.lua]: @istoimenova, Agree, but in script activation is linked with PTU and required in order to have table from snapshot created

**Pull request merged**